Flow/README.md

```diff
@@ -0,0 +1,25 @@
+# Flow
+
+## Screenshots
+
+### Ocean
+<img src="https://raw.githubusercontent.com/ossd-sp22/spicetify-themes/newTheme/Flow/screenshots/ocean.png" alt="img"> 
+
+
+### Pink
+<img src="https://raw.githubusercontent.com/ossd-sp22/spicetify-themes/newTheme/Flow/screenshots/pink.png" alt="img"> 
+
+
+### Silver
+<img src="https://github.com/ossd-sp22/spicetify-themes/blob/newTheme/Flow/screenshots/silver.png" alt="img"> 
+
+
+### Violet
+<img src="https://raw.githubusercontent.com/ossd-sp22/spicetify-themes/newTheme/Flow/screenshots/violet.png" alt="img"> 
```

Comment:
@Ruixi-Zhang This flew by but can you make another PR to change these to use the spicetify/spicetify-themes and the main branch as well?

Reply:
@afonsojramos The changes were updated by @ian-Liaozy